orchagent/portsorch.cpp

```diff
@@ -965,6 +965,7 @@ bool PortsOrch::initializePort(Port &p)
 #endif
 
     /* Set default port admin status to DOWN */
+    /* XXX: Do we need this? The default port admin status is false */
```

Comment:
[https://softwareengineering.stackexchange.com/questions/65467/what-does-xxx-mean-in-a-comment](https://softwareengineering.stackexchange.com/questions/65467/what-does-xxx-mean-in-a-comment)

---
In reply to: [126577048](https://github.com/Azure/sonic-swss/pull/250#discussion_r126577048) [](ancestors = 126577048)